### PR TITLE
Answers Week1

### PR DESCRIPTION
**Pull request by DYA13**

### CODE REVIEW

File: 01-node-tutorial/answers/app.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+res.end(`
+<h1>Oops!</h1>
+<p>We can't seem to find the page you are looking for</p>
+<a href="/"> back home</a>
+`)
+})
+
+server.listen(9000)
+
+
```
Comment (akosasante) [inline 01-node-tutorial/answers/app.js:19]: This looks good, just make sure to move this content to a file named `12-http.js` as we will keep adding new modules in the next week's lesson.

**Review by akosasante — APPROVED** [review]
Great job, everything looks great!

Just a couple small comments before you go ahead and merge this PR:

*   I notice there are a few `.DS_Store` files in this PR. These do not need to be commited as they are not part of the code. It is just a file that is sometimes created automatically on [Apple/MacOS computers.](https://en.wikipedia.org/wiki/.DS_Store)   
      
    You can tell `git` to ignore these files and not add them to your commits by adding a file named `.gitignore` to your `node-express-course` directory. In that file, just have a line that says `.DS_Store`. The `git` program reads that file to know what file names or directories it should completely ignore when you're doing `git` commands like `git add` or `git commit`.
*   One thing I would recommend, is if you're using Visual Studio Code, you can go to the "View" menu at the top toolbar, and use the "Command Palette" to run the "Format Document" command; which should automatically add spaces and line breaks so that your javascript code is well-formatted in a standard way (eg: instead of `const  amount= 12;`, should have only one space after `const` like this: `const amount = 12;`).   
      
    You can also take a look at the extension ["Prettier"](https://www.digitalocean.com/community/tutorials/how-to-format-code-with-prettier-in-visual-studio-code) which can be installed in VS Code to help format Javascript files with particular formatting options.  
     

Feel free to comment in the PR or let me know in the CodeTheDream Slack if you have any questions!

**Comment by akosasante** [comment]
Actually, @DYA13 I notice that this PR is on the `Code-the-Dream-School` repo instead of your own repo:



*   You'll want to go to _your_ forked repo: [https://github.com/DYA13/node-express-course](https://github.com/DYA13/node-express-course)
*   Go to the `Pull Requests` tab
*   Make sure the `base` repo and branch on the left are set to your repo `DYA13/node-express-course` and the `main` branch
*   Make sure the `head_repository` and `compare` branch on the right are set to your repo `DYA13/node-express-course` and the `week-1` branch
*   The green button should then say something like "Create Pull Request", click that and if you're ready to merge (have taken a look at my above comments about `.DS_Store` and formatting), then go ahead and merge that PR. 
*   For this PR you can just "Close PR" without merging. The idea is to keep the Code-the-Dream-School version of this repository as just the base that all students can fork from.

**Comment by jrmcgarvey** [comment]
Directed to wrong repository.

**Pull request closed without merging**